great_expectations/profile/user_configurable_profiler.py

```diff
@@ -202,6 +201,7 @@ def build_suite(self):
 
         """
         if len(self.profile_dataset.get_expectation_suite().expectations) > 0:
+            # noinspection PyProtectedMember
```

Comment:
Why was this added?

Reply:
To prevent warnings about using private methods.